pkg/detectors/jdbc/jdbc.go

```diff
@@ -5,6 +5,7 @@ import (
 	"database/sql"
 	"errors"
 	"fmt"
+	logContext "github.com/trufflesecurity/trufflehog/v3/pkg/context"
```

Comment:
nit: Could we move this import below the std library pkgs.

Reply:
Done.